docs/src/knownissues.md

```diff
@@ -129,8 +152,11 @@ Make sure to:
 
 After that, [this script](https://gist.github.com/luraess/c228ec08629737888a18c6a1e397643c) can be used to verify if ROCm-aware MPI is functional (modified after the CUDA-aware version from [here](https://discourse.julialang.org/t/cuda-aware-mpi-works-on-system-but-not-for-julia/75060/11)). It may be preferred to run the Julia ROCm-aware MPI script launching it from a shell script (as suggested [here](https://discourse.julialang.org/t/cuda-aware-mpi-works-on-system-but-not-for-julia/75060/4)).
 
-## Microsoft MPI
+## Custom reduction operators
 
-### Custom operators on 32-bit Windows
+It is not possible to use custom reduction operators [with 32-bit Microsoft MPI](https://github.com/JuliaParallel/MPI.jl/issues/246) on Windows and on [ARM CPUs](https://github.com/JuliaParallel/MPI.jl/issues/404) with any operating system.
```

Comment:
```suggestion
It is not possible to use custom reduction operators [with 32-bit Microsoft MPI](https://github.com/JuliaParallel/MPI.jl/issues/246) on Windows and on [ARM or PowerPC CPUs](https://github.com/JuliaParallel/MPI.jl/issues/404) with any operating system.
```

Reply:
My understanding is that this now works on PowerPC.  @vchuravy?

Reply:
See https://github.com/JuliaLang/julia/issues/32154, now closed as resolved.

Reply:
Ah, good to know!